app/src/components/common/__tests__/Table.spec.ts

```diff
@@ -100,4 +109,76 @@ describe('Table.vue', () => {
     await wrapper.setProps({ data: [] })
     expect(wrapper.find('tbody p').exists()).toBe(true)
   })
+
+  it('renders sorted data when clicking on sortable column header', async () => {
+    //ascending sorting option
+    await wrapper.find('thead th:nth-child(2)').trigger('click')
+    expect(wrapper.findAll('tbody tr').at(0).text()).toContain(
+      data[ 1 ].age.toString()
+    )
+    expect(wrapper.findAll('tbody tr').at(1).text()).toContain(
+      data[ 0 ].age.toString()
+    )
+    //descending sorting option
+    await wrapper.find('thead th:nth-child(2)').trigger('click')
```

Comment:
There is no need to define the `thread th:nth-child(2)` element each time you want to trigger a click. You can just define it only once and you it in the scope of your test:

const component = wrapper.find('thead th:nth-child(2)')
component..trigger('click')

Reply:
Fixed it in the recent commit.